testsuite/features/support/client_stack.rb

```diff
@@ -85,3 +80,19 @@ def get_os_version(node)
 
   [os_version, os_family]
 end
+
+def sle11family?(node)
```

Comment:
On the case of sle**family?(node) is a readable function, but for opensuse?(node), it could lead to confusion, I would have all of them with the same format is****?(node) or ****family?(node)

Reply:
ok - but this is work in progress, and this function might simply go. It seems it's not even needed.

Reply:
Function gone